cluster-controller/src/main/java/io/strimzi/controller/cluster/ClusterControllerConfig.java

```diff
@@ -28,22 +66,63 @@ public static ClusterControllerConfig fromEnv() {
             labelsMap.put(fields[0].trim(), fields[1].trim());
         }
 
-        return new ClusterControllerConfig(namespace, labelsMap);
+        return new ClusterControllerConfig(namespace, labelsMap, reconciliationInterval);
     }
 
+    /**
+     * @return  labels used for watching the cluster ConfigMap
+     */
     public Map<String, String> getLabels() {
         return labels;
     }
 
+    /**
+     * Set the labels used for watching the cluster ConfigMap
+     *
+     * @param labels    labels used for watching the cluster ConfigMap
+     */
     public void setLabels(Map<String, String> labels) {
         this.labels = labels;
     }
 
+    /**
+     * @return  namespace in which the controller runs and creates resources
+     */
     public String getNamespace() {
         return namespace;
     }
 
+    /**
+     * Set the namespace in which the controller runs and creates resources
+     *
+     * @param namespace namespace in which the controller runs and creates resources
+     */
     public void setNamespace(String namespace) {
```

Comment:
No one actually uses any of these setters, so we could remove them (different PR).